spec/routes/web/auth_spec.rb

```diff
@@ -461,7 +475,21 @@ def mock_provider(provider, email = TEST_USER_EMAIL)
         expect(page).to have_content "Your account has been disconnected from Github"
       end
 
-      it "can not disconnect the last login method" do
+      it "can delete password if another login method is available" do
+        account.add_identity(provider: "google", uid: "123456790")
+        account.add_identity(provider: "github", uid: "123456790")
```

Comment:
May want to only have a single provider for this test, since you should be able to delete with a single provider.